log_outgoing_requests/static/log_outgoing_requests/css/admin.css

```diff
@@ -0,0 +1,6 @@
+.field-res_body {
+	display: inline-block;
+	width: 40rem;
+	font-family: monospace;
+	line-height: 1.5rem;
+}
```

Comment:
please use two spaces instead of a tab character, and update the `.editorconfig` to include this formatting rule